entity-types/infra-confluentcloudcluster/definition.yml

```diff
@@ -34,25 +34,25 @@ dashboardTemplates:
 synthesis:
   rules:
   # telemetry from opentelemetry provider
-  - identifier: kafka_id
-    name: kafka.cluster_name
-    encodeIdentifierInGUID: true
-    tags:
-      kafka_id:
-      kafka.cluster_name:
-      instrumentation.provider:
-      newrelic.source:
-    conditions:
-      - attribute: eventType
-        value: Metric
-      - attribute: metricName
-        prefix: confluent_kafka_server_
-      - attribute: instrumentation.provider
-        value: opentelemetry
-      - attribute: kafka_id
-        present: true
-      - attribute: topic
-        present: false
+  # - identifier: kafka_id
```

Comment:
Any specific reason to comment out data? If we plan to re-introduce it later we can just revert the commit